### PR TITLE
NotificationServer: Fix the layout for multi-line notifications

### PR DESCRIPTION
**Pull request by AtkinsSJ**
**Before:**

[Peek 2024-01-16 16-15 broken notifications.webm](https://github.com/SerenityOS/serenity/assets/222642/be5dd8c7-ec75-4c5b-aec6-ce1c29d36772)

**After:**

[Peek 2024-01-16 16-06 fixed notifications.webm](https://github.com/SerenityOS/serenity/assets/222642/6522f9ca-5688-44da-b7e6-406478f83bc8)

It's a little hacky to manually calculate the heights here, but it works, until the layout system can handle this better.

### CODE REVIEW

**Comment by AtkinsSJ** [comment]
:thinking: I just noticed in #9441 that the margins are much smaller. Did we accidentally increase them at some point?

Also whoops, I didn't realise I was recording my desktop audio in those videos. :laughing:

**Comment by AtkinsSJ** [comment]
Changes:
- Rebase to fix a conflict.
- The "GMLCompiler: Add support for the font-weight enum values" commit is now just sorting the enums list and removing a duplicate, because another commit has added the font-weight enum already.

File: Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp [hunk 1]
```diff
@@ -354,6 +353,7 @@ static ErrorOr<String> generate_cpp(NonnullRefPtr<GUI::GML::GMLFile> gml, Lexica
         TRY(String::from_utf8("<AK/JsonValue.h>"sv)),
         TRY(String::from_utf8("<AK/NonnullRefPtr.h>"sv)),
         TRY(String::from_utf8("<AK/RefPtr.h>"sv)),
+        TRY(String::from_utf8("<LibGfx/Font/FontWeight.h>"sv)),
```
Comment (trflynn89) [inline Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp:356]: Adding this include here seems unrelated to this commit, was it meant to be squashed into the previous one?

Comment (AtkinsSJ) [reply Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp:356]: Ah I missed this when rebasing. Originally this commit added FontWeight to the compiler so this made more sense!

**Comment by AtkinsSJ** [comment]
Change: Moved the GML Compiler `<LibGfx/Font/FontWeight.h>` include into the commit that creates that header.

**Review by trflynn89 — APPROVED** [review]
